spring-cloud-function-samples/function-sample-gcp/pom.xml

```diff
@@ -2,69 +2,57 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
 		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
 	<modelVersion>4.0.0</modelVersion>
 
+	<groupId>io.spring.sample</groupId>
 	<artifactId>function-sample-gcp</artifactId>
+	<version>2.0.0.RELEASE</version>
+	<packaging>jar</packaging>
+
+	<name>function-sample-gcp</name>
 
 	<parent>
-		<artifactId>spring-cloud-function-samples</artifactId>
-		<groupId>org.springframework.cloud</groupId>
-		<version>3.1.0.BUILD-SNAPSHOT</version>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
```

Comment:
Is this OK? It will now deviate from the other samples.

Reply:
I believe the other samples actually inherit from `spring-boot-starter-parent`, so this is will make our sample consistent with others.

Reply:
oh! :)